core/contracts/common/implementation/MultiRole.sol

```diff
@@ -35,6 +35,7 @@ library Shared {
     }
 
     function addMember(RoleMembership storage roleMembership, address memberToAdd) internal {
+        require(memberToAdd != address(0x0));
```

Comment:
I would make this the only require without a message and add a comment above it pointing to the issue and commenting the require message that we'd *like* to have here once the compilation issue is fixed.

Reply:
Wait, sorry, is this the one that breaks it? If not, can you add an error message back and move the above comment to the one that actually breaks the compilation?